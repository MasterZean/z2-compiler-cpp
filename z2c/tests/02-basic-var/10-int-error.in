val a = -2147483648
val b = 2147483648
val c = -3000000000
val d = 3000000000