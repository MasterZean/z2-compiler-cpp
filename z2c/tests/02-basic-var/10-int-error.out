int32 a = (int32)-2147483648ll;
DummyClass(2, 9): error:
	'Long' literal constant requires 'l' sufix
DummyClass(3, 9): error:
	'Long' literal constant requires 'l' sufix
DummyClass(4, 9): error:
	'Long' literal constant requires 'l' sufix
